src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -250,6 +253,10 @@ nuget list -verbose -allversions</value>
   <data name="ListCommandVerboseListDescription" xml:space="preserve">
     <value>Displays a detailed list of information for each package.</value>
   </data>
+  <data name="Log_RestoreNoCacheInformation" xml:space="preserve">
+    <value>The NoCache option is only for disabling the HTTP cache and not the global packages folder. It is recommended to use the NoHttpCache option instead to disable the use of the HTTP cache.</value>
```

Comment:
We're probably gonna duplicate these, but I'm leaving feedback on one.

IMO, we don't need to be this verbose, we can just say `NoCache` is deprecated in favor of the appropriately named `NoHttpCache`, or something along those lines. 

Curious what others on the team think.

Reply:
Alternatively, "NoCache has been renamed to NoHttpCache. The behaviour remains the same".

Unfortunately my suggestion might cause some customers to contact us asking us if this means that NoHttpCache ignores other "caches", and we'll have to inform them no, NoCache was always only the http cache.

If we're concerned about that, we could try "NoCache has been renamed to NoHttpCache, as NoCache was often misunderstood to behave differently."